src/main/java/com/google/cloud/tools/appengine/api/deploy/DefaultGenRepoInfoFileConfiguration.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Since this is going to call a command under 'gcloud debug', and conceivably tools can call in to this at build time, should we put this in its own package?